code/camera/cameradisplay.cpp

```diff
@@ -63,16 +63,14 @@ CameraDisplay::CameraDisplay(QWidget *parent) :
     populate_effect_box(m_ui->effect_box);
 
     // Setup zoom slider
-    m_ui->zoom_slider->setTickInterval(Zoom::ZOOM_INTERVAL);
-    m_ui->zoom_slider->setTickPosition(QSlider::TicksBelow);
-    m_ui->zoom_slider->setMaximum(Zoom::ZOOM_MAX);
-    m_ui->zoom_slider->setMinimum(Zoom::ZOOM_MIN);
+    setup_slider(*(m_ui->zoom_slider), 10, Zoom::ZOOM_MIN, Zoom::ZOOM_MAX, Zoom::ZOOM_INTERVAL, true);
 
     // Setup rotation slider
-    m_ui->rotate_slider->setTickInterval(Rotation::ROTATION_INTERVAL);
-    m_ui->rotate_slider->setTickPosition(QSlider::TicksBelow);
-    m_ui->rotate_slider->setMaximum(Rotation::ROTATION_MAX);
-    m_ui->rotate_slider->setMinimum(Rotation::ROTATION_MIN);
+    setup_slider(*(m_ui->rotate_slider), 0, Rotation::ROTATION_MIN, Rotation::ROTATION_MAX, Rotation::ROTATION_INTERVAL, true);
+
+    // Setup griddisplay sliders
+    setup_slider(*(m_ui->horizontal_grid_slider), 0, GridLocation::GRIDLOCATION_MIN, GridLocation::GRIDLOCATION_MAX, GridLocation::GRIDLOCATION_INTERVAL, true);
```

Comment:
These lines are a bit long 
Can you keep them under 150 characters ish

Reply:
Just put in newlines? The actual code itself is fine right?

Reply:
yeah like
```c++
function_call(
    argument1, argument2, argument3,
    argument4, agument5, argument6
);
```

Reply:
This is why text editors have word wrap tho :cry: